3 - PB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -944,18 +944,25 @@ I documenti pertanto sono così strutturati:
 + *Contenuto del file*: sezione successiva agli indici. Rappresenta il corpo del documento, suddiviso in paragrafi.
 
 ==== Struttura dei verbali <struttura-verbali>
-I verbali assumono una struttura diversa rispetto agli altri documenti, dato il diverso scopo e la struttura semplificata. I verbali sono così strutturati:
-
-- *cover page* (@struttura-documenti);
-- *informazioni generali*:
-  - luogo;
+Il gruppo ha concordato sulla necessità di tenere traccia dei meeting interni ed esterni svolti, per questo vengono redatti dei verbali che devono essere precisi e completi nel riportare i punti fondamentali degli incontri.\
+Per soddisfare i precedenti parametri è stata definita una struttura per aiutare il redattore nella scrittura del documento:
++ *cover page* (@struttura-documenti);
++ paragrafo "*informazioni generali*":
+  - luogo: il luogo reale o digitale dove il gruppo insieme ad eventuali partecipanti esterni svolge il meeting;
   - data e ora nel formato (gg-mm-aaaa, hh:mm ~ hh:mm);
   - partecipanti;
   - assenti;
-  - referente aziendale (se presente).
-- *ordine del giorno*: elenco degli argomenti trattati durante la riunione;
-- *organizzazione attività*: elenco e spiegazione delle decisioni prese durante la riunione. Questo paragrafo rappresenta il risultato fondamentale delle riunioni di retrospettiva;
-- *firma partecipanti esterni* (se presenti): firma dei partecipanti esterni alla riunione.
+  - partecipanti esterni al gruppo (se presenti).
++ paragrafo "*ordine del giorno*": elenco degli argomenti trattati durante la riunione;
++ paragrafo "*Valutazione del progresso generale*" descrive l'analisi sul lavoro svolto durante lo sprint precedente. Questo paragrafo è così definito:
+  - valutazione sulla qualità del lavoro in merito alla velocità di avanzamento, evidenziando eventuali rallentamenti o meriti riscontrati.
+  - sottoparagrafi per ogni epic su cui si è lavorato in cui viene riportato una descrizione sul lavoro svolto, se i lavori svolti sono molti e diversi riportarli in forma di elenco puntato.
++ paragrafo "*Analisi di retrospettiva*", suddiviso in:
+  - tabella che riporta i valori calcolati dalle metriche presenti nel #pdq riguardo allo sprint, nello specifico riportare le metriche: task completate, SEV, SPV, SAC.
+  - sottoparagrafo "*Keep doing*", riporta i meriti riscontrati dal gruppo;
+  - sottoparagrafo "*Improvements*", riporta una serie di criticità riscontrate dal gruppo etichettate con un codice identificativo del tipo C#emph[i] con #emph[i] numero naturale positivo. Quindi una riportare una tabella che associa ad ogni criticità titolo e codice del metodo di risoluzione contenuto nel #pdp.
++ eventuali altri paragrafi che descrivono eventi di cui si ritiene importante la documentazione.
```

Comment:
"eventuali ... documentazione" -> eventuali ulteriori argomenti organizzati in paragrafi indipendenti.